package.json

```diff
@@ -6,14 +6,20 @@
   "scripts": {
     "build-css": "./css-build",
     "check-coverage": "istanbul check-coverage --config .istanbul.yml",
+    "check-unit-coverage": "istanbul check-coverage --config .istanbul.unit.yml",
     "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
-    "git-hook": "yarn run lint && yarn run generate-coverage && yarn run check-coverage && yarn run snyk -- test",
+    "generate-unit-coverage": "yarn run istanbul -- cover _mocha -- --recursive test/unit",
+    "git-hook": "yarn run lint && yarn run generate-unit-coverage && yarn run check-unit-coverage",
```

Comment:
I'll add the subset info in the readme - the integration tests run in the container and rely on ES.